components/product-list.html

```diff
@@ -0,0 +1,64 @@
+<div class="bg-white">
+    <div class="  bg-gray-100 max-w-4xl mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:max-w-7xl lg:px-8">
+      <h2 class="text-2xl text-gray-900 font-extrabold tracking-tight ">Product-List</h2>
+      <div class="mt-6 grid grid-cols-1 gap-y-10 gap-x-6 sm:grid-cols-2 lg:grid-cols-4 xl:gap-x-8">
+           <!--Product-1 -->
+        <div class="group relative">
+          <div class="w-full min-h-90 bg-gray-200 aspect-w-1 aspect-h-1 rounded-lg overflow-hidden group-hover:opacity-75 lg:h-80 lg:aspect-none">
+            <img src="https://5.imimg.com/data5/YX/OO/TA/ANDROID-108727015/product-jpeg-500x500.jpg" alt="Men Navy-blue Tshirt#9087" class="w-full h-full object-center object-cover lg:w-full lg:h-full">
+          </div>
+          <div class="mt-4 flex justify-between">
+              <div>
+              <p class="mt-1 text-sm text-gray-500">Navy-Blue</p>
+              </div>
+              <div>
+              <p class=" mt-1 text-sm font-medium text-gray-900">$25</p>
+              </div>
```

Comment:
```suggestion
              <p class="mt-1 text-sm text-gray-500">Navy Blue</p>
              <p class=" mt-1 text-sm font-medium text-gray-900">$25</p>
```

The extra `<div>` tags aren't necessary, the paragraph tags are enough.

Reply:
Ok . Thank you for suggestion. I will work on it